bedrock/foundation/templates/foundation/index.html

```diff
@@ -13,89 +13,128 @@ <h1 class="page-title">{{ _('The Mozilla Foundation') }}</h1>
 {% endblock %}
 
 {% block article %}
+  <h2>{{ _('We Are Mozilla') }}</h2>
+
+  <p>
+    {% trans url_manifesto=url('mozorg.about.manifesto') %}
+      The nonprofit Mozilla Foundation believes the internet must always remain a global public resource that is open and accessible to all. Our work is guided by the <a href="{{ url_manifesto }}">Mozilla Manifesto</a>.
+    {% endtrans %}
+  </p>
+
+  <p>
+    {% trans %}
+      The direct work of the Mozilla Foundation focuses on making the health of the internet a mainstream issue. We do this by connecting open internet leaders with each other and by mobilizing grassroots activities around the world.
+    {% endtrans %}
+  </p>
+
   <p>
     {% trans %}
-      The nonprofit Mozilla Foundation is the organizational home of the
-      Mozilla Project, a global community and public interest initiative
-      that believes the Web should be open and accessible to all. To
-      protect the Web as a public resource and empower its users, we
-      create open source products, teach 21st-century skills and spur
-      grassroots advocacy campaigns. All this is made possible by
-      full-time staff and thousands of volunteers around the world.
+      The Foundation is also the sole shareholder in the Mozilla Corporation, the maker of Firefox and other open source tools. Mozilla Corporation functions as a self-sustaining social enterprise -- money earned through its products is reinvested into the organization. 
     {% endtrans %}
   </p>
 
+  <h2>{{ _('Foundation Programs') }}</h2>
+
   <p>
-    {% trans firefox=url('firefox.new') %}
-      To keep the Web accessible, we build products that create
-      opportunity and inspire further innovation. Many of Mozilla’s
-      products are developed by the Mozilla Corporation, a wholly-owned
-      subsidiary of the Mozilla Foundation. The Mozilla Corporation
-      functions as a self-sustaining social enterprise — money earned
-      through its products is reinvested into the organization. Products
-      include our flagship <a href="{{ firefox }}">Firefox browser</a> and
-      other open source tools. The Mozilla Corporation also partners with
-      like-minded allies to bring an open source ethos to industries and
-      technologies like online video, telecommunications and virtual reality.
+    {% trans %}
+      Working with people and communities all across Mozilla, the Foundation team is focused on fueling the open internet movement. Our programs are focused in three areas:
     {% endtrans %}
   </p>
 
+  <ol>
```

Comment:
Now that I think about it, maybe an ordered list isn't really best for this copy, just putting numbers in the headings might be better: `<h3>{{ _('1. Shape the agenda') }}</h3>` ... what do you think?